src/components/UI/Pager/Pager.tsx

```diff
@@ -192,13 +202,27 @@ export const Pager = ({
 
   return (
     <div className={styles.TableContainer}>
-      <Table className={styles.Table} data-testid="table">
-        <TableHead data-testid="tableHead">{tableHead}</TableHead>
-        <TableBody data-testid="tableBody">{rows}</TableBody>
-        <TableFooter className={styles.TableFooter} data-testid="tableFooter">
-          <TableRow>{tablePagination}</TableRow>
-        </TableFooter>
-      </Table>
+      <TableContainer
+        sx={{
```

Comment:
Let's avoid using sx and use modular css to be consistent

Reply:
Resolved